doc/crypto/api/keys/management.rst

```diff
@@ -216,7 +254,7 @@ When creating a key, the attributes for the new key are specified in a `psa_key_
     .. retval:: PSA_ERROR_BAD_STATE
         The library requires initializing by a call to `psa_crypto_init()`.
 
-    Copy key material from one location to another.
+    Copy key material from one location to another. Its location is taken from ``attributes``, its policy is determined by both ``attributes`` and the source key policy, and its type and size are taken from the source key.
```

Comment:
The description of the resulting key policy is vague and could be read as e.g. the union of the two policies rather than the intersection. The policy is _constrained_ by both `attributes` and the source key policy, although that may not be a comprehensible way to put it. In plain language, the resulting key can do everything that is both permitted for the original key and permitted by `attributes`.

Reply:
This first paragraph is aiming to be a summary of how the attributes are derived (cf the other key creation functions), and the full detail regarding policies is provided a few paras later in this description.

I'd rather not bloat this sentence, but would be happy to find a short phrase that more accurately describes the combination/intersection of policies for use here, that reduces the misunderstanding of the process.